.github/workflows/eclipse-publish.yml

```
@@ -0,0 +1,135 @@
+permissions:
+  contents: write
+  id-token: write
+
+concurrency:
+  group: release-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  release:
+    name: Release to npm
+    if: startsWith(github.event.head_commit.message, 'chore(eclipse): release ')
```

actionlint (1.7.11)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)